dev/io.openliberty.org.apache.myfaces.4.0_fat/fat/src/io/openliberty/org/apache/myfaces40/fat/tests/AnnotationLiteralsTest.java

```diff
@@ -0,0 +1,357 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
```

Comment:
As far as I know you can drop the `contributors` section in the new copyright header for EPL-2.0

Reply:
The Eclipse template still adds it so I'm going to keep it until that is updated. 
I think they plan to go through and remove them as a batch.